test/verifiedTests/testCardOpt/testSparseLP.m

```diff
@@ -6,18 +6,13 @@
 % Authors:
 %     - CI integration: Laurent Heirendt March 2017
 %
-% Note:
-%     - The solver libraries must be included separately
-
-% define global paths
-global path_TOMLAB
-global path_GUROBI
 
 % save the current path
 currentDir = pwd;
 
 % initialize the test
-initTest(fileparts(which(mfilename)));
+fileDir = fileparts(which('testSparseLP'));
```

Comment:
The simple answer is no. 

`R2014b` and `R2015b` do not support this when the tests are run in `runtests()`. This is the only way this can work on `R2014b` and `R2015b` when the tests are called in `runtests()`. Reason is that `mfilename` is the to the path of `ScriptTestCaseProvider` in `runtests`.